### DISCUSSION TITLE
Feature Request: Option to add an "extended presence" binary sensor

### DISCUSSION THREAD

**Discussion by Nuuki9** [jseidl/magic-areas#448] [Ideas / Feature Requests]
The ability to trigger actions when extended presence is triggered is great. However aside from controlling devices within MA itself, I'd also like to be able to use that to trigger automations. Unless I'm missing something `extended` is only provided through the `States` attribute, but testing for an attribute containing a defined string is tricky to test for - I think you need to use a template condition.

Would it be possible to add an option to create a second binary sensor `Area Extended State`?

**Comment by jseidl** [comment]
Yeah template condition should work fine. The attribute isn't a string but a list. It just gets rendered as a comma-separated stringy-thing.

![image](https://github.com/user-attachments/assets/63fd0600-5ff8-4afe-a055-23a62a720240)

Since it's simple to automate on it, I'm don't see much value adding a binary sensor for each secondary state. If you really need to be a sensor you can create a template binary sensor based off the attribute.

Is there a specific reason that the above wouldn't work?
